src/FsAutoComplete/FsAutoComplete.fsproj

```diff
@@ -45,7 +45,7 @@
     <PackageType>DotnetTool</PackageType>
     <!-- This rollforward enables us to actually run on the .net 6 runtime without
          retargeting the whole app, minimizing packaged binary size. -->
-    <RollForward>Major</RollForward>
+    <RollForward>LatestMajor</RollForward>
```

Comment:
Unsure if this is actually needed

Reply:
nah, in fact because we _do_ multitarget the app now we could remove the whole thing